script.js

```diff
@@ -14,3 +14,199 @@ mobileLinks.forEach((link) => link.addEventListener('click', () => {
+        <li class="year">${worksInfo[index].year}</li>
+      </ul>
+      <p class="work-info">${worksInfo[index].description}</p>
+      <ul class="tags">
+        <li class="html tag">${worksInfo[index].htmlBadge}</li>
+        <li class="ruby tag">${worksInfo[index].rubyBadge}</li>
+        <li class="css tag">${worksInfo[index].cssBadge}</li>
+        <li class="js tag">${worksInfo[index].jsBadge}</li>
+      </ul>
+      <button class="btn btn-project" id="${[index]}">${worksInfo[index].buttonText}</button>
+    </div>
+  </div>
+ `;
+});
+
+// modal
+const seeProject = document.querySelectorAll('.btn-project');
+seeProject.forEach((element) => {
+  // open and display modal when you click on project button
+  element.addEventListener('click', () => {
+    const modalContainer = document.createElement('section');
+    modalContainer.classList.add('modal-container');
+    modalContainer.innerHTML = `
+    <section class="modal">
+       
+        <div class="modal-header">
+          <h3>${worksInfo[element.id].workTitle}</h3>
+          <img class="close" src="images/close.svg" alt="">
+        </div>
+        <ul class="job">
+          <li class="company">${worksInfo[element.id].company}</li>
+          <li class="counter"><img src="images/counter.png" alt=""></li>
+          <li class="job-title">${worksInfo[element.id].jobTitle}</li>
+          <li class="counter"><img src="images/counter.png" alt=""></li>
+          <li class="year">${worksInfo[element.id].year}</li>
+        </ul>
+        <div class="modal-image">
+        <img class="modal-img mobile" src="${worksInfo[element.id].workImgMobile}" alt="">
+        <img class="modal-img desktop" src="${worksInfo[element.id].workImgDesktop}" alt="">
+      </div>
+        <div class="modal-card-text">
+          <p class="modal-work-info">${worksInfo[element.id].modalDescription}</p>
+          <div class="tags-buttons">
+            <ul class="modal-tags">
+              <li class="modal-html tag">${worksInfo[element.id].htmlBadge}</li>
+              <li class="modal-css tag">${worksInfo[element.id].cssBadge}</li>
+              <li class="modal-js tag">${worksInfo[element.id].jsBadge}</li>
+            </ul>
+            <ul class="modal-tags">
+              <li class="modal-gh tag">${worksInfo[element.id].ghBadge}</li>
+              <li class="modal-ruby tag">${worksInfo[element.id].rubyBadge}</li>
+              <li class="modal-bootstrap tag">${worksInfo[element.id].bootstrapBadge}</li>
+            </ul>
+            <div class="modal-buttons">
+              <a class="live-link" href=""><button class="live modal-btn">See Live <img class="btn-image" src="images/live.png" alt="Live Preview"></button></a>
+              <a class="source-link" href=""><button class="source modal-btn">See Source <img class="btn-image" src="images/source.svg" alt="Source Code"></button></a>
+            </div>
+          <div>
+        </div>
```

Comment:
- [ ] Good job implementing the details modal in desktop view. Kindly make sure that it's responsive on mobile view to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/168084317-419f2824-75e7-49ba-bc14-3f1c73d579d4.png) | ![image](https://user-images.githubusercontent.com/62290545/168084272-afdc2ee6-1d8a-4a19-af9f-3a545a42215d.png) |